lib/IR/Value.cpp

```diff
@@ -33,6 +33,7 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/ManagedStatic.h"
 #include "llvm/Support/raw_ostream.h"
+#include <exception>
```

Comment:
Please add `// HLSL Change` to mark this as changed by DXC.

Reply:
Oh right, this. done thanks!